packages/core/src/actions/getTransactionCount.test.ts

```diff
@@ -0,0 +1,66 @@
+import { accounts, chain, config, testClient } from '@wagmi/test'
+import { describe, expect, test } from 'vitest'
+
+import { getTransactionCount } from './getTransactionCount.js'
+
+const address = accounts[0]
+
+describe('getTransactionCount', () => {
```

Comment:
Let's remove the `describe`. We prefer to not use it in the test suite unless there's a reason to do a lot of grouping.

Reply:
makes sense, in this case for consistency I'd suggest also removing it for [`getBalance`](https://github.com/wevm/wagmi/blob/beta/packages/core/src/actions/getBalance.test.ts#L22) too.